src/main/java/com/devonfw/quarkus/ordermanagement/service/v1/model/NewOrderDto.java

```diff
@@ -0,0 +1,19 @@
+package com.devonfw.quarkus.ordermanagement.service.v1.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Getter;
+import lombok.Setter;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import java.util.Date;
+import java.util.List;
+
+@Getter
+@Setter
+public class NewOrderDto {
+    @Schema(nullable = false, description = "Payment date in format dd-MM-yyyy")
+    @JsonFormat(pattern = "dd-MM-yyyy")
+    private Date paymentDate;
```

Comment:
This is inconsistent: See `OrderDto` that is using `Instant` for the same property. BTW: java.util.Date and Caldendar are deprecated and should never be used anymore.

Is this really a best practice to have `NewOrderDto` and `OrderDto`?
IMHO we are getting a little bit off track here:
* Either we properly apply command pattern
* Or we go for a simple TO pattern and should IMHO consider reuse. But I also see the point that in this special case for order `create` and `update` rather differ so I would like to hear feedback from others as well... If we want to keep this here, should we add something to the documentation about reuse TOs vs. crafting separate redundant TOs for individual use-cases/operations on the same entity...

Reply:
I understand what you mean. But in this example, the only property is paymentDate, which is a duplicate. So I would leave it as it is for now and adapt the documentation accordingly to the reuse of TOs.

Reply:
Yes, fully agree. Thanks 👍